console/account/how-to/resolve-abuse.mdx

```diff
@@ -0,0 +1,62 @@
+Scaleway’s clients are responsible for taking the appropriate security measures in order to solve the case and prevent any further abuse from their resources in order to avoid the locking of their services.
+However, Scaleway reserves the right to immediately suspend the infringing services if it is deemed necessary and proportionate to keep Scaleway’s network safe or to abide by legal requirements.
+
+If you encounter any issue with the resolution of an abuse, please contact the abuse team by opening a [support ticket](https://console.scaleway.com/support/tickets/create) directly from your console.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+</Message>
+
+1. Click **Abuse** in the bottom left corner of the Scaleway console. A list of your open cases displays. You can also view the closed cases by clicking on “Closed cases”.
+2. Click the **case ID** to view the specifics of the complaint.
+3. Take note of the details provided and proceed with the necessary steps to address and resolve the issue. 
+    <Message type="tip">
+      If you need guidance on resolving outgoing DDoS attacks from your server, refer to our documentation [how to prevent outgoing DDoS](https://www.scaleway.com/en/docs/compute/instances/reference-content/preventing-outgoing-ddos/) for assistance.
+    </Message>
+    In case you receive an abuse report for hosting illegal content (including copyright), please delete the content from Scaleway infrastructure promptly.
+    * For other types of abuse, here are some suggestions to help you to correct them:
+        * Backup all your server data
+        * Check for recent file changes
+        * Check log files on your server
+        * Check your PHP and cgi files for any backdoors
+        * Check if any software you use is vulnerable, and update them
+        * Reinstall your server after backup
+        * Change passwords of your server
+        * Backup your website and your email data
+        * Check for recent files changes, security holes in common CMS you may use (Joomla, SPIP, WordPress…)
+        * Check version in some software you may use on your computer (email client, FTP client)
+        * Change your FTP and email passwords
+    If you do not know how to correct the problem, you can contact our [outsourcing partners(https://www.scaleway.com/en/dedibox/outsourcing/)] or reach out to Scaleway’s abuse team by opening a ticket directly from your console.
+
+4. The case will be closed at the end of its lifetime by Scaleway’s abuse team, and so will the abuse tickets forming the case.
+   In case you receive further abuse reports on the same resource within the case lifetime, it means that you did not take the appropriate measures to prevent further abuse from happening. 
```

Comment:
```suggestion
   If you receive more abuse reports on the same resource within the case lifetime, it means that you did not take the appropriate measures to prevent further abuse from happening. 
```